guides/live_view.md

```diff
@@ -120,6 +123,7 @@ LiveViews have several important lifecycle stages:
 
 - [`mount`](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#c:mount/3) - initializes the LiveView with parameters, session data, and socket
 - [`handle_params`](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#c:handle_params/3) - responds to URL changes and updates LiveView state accordingly
+- [`render`](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#c:render/1) - renders the LiveView's HTML based on the current state
```

Comment:
After thinking about it, I would not mention render here. While it is technically part of the lifecycle, it is very different to the other stages mentioned here and people will read more about it in the LiveView documentation. It can also be omitted in favor of a html.heex file.